### PR TITLE
Updating instructions on screenshots.

### PR DESCRIPTION
**Pull request by dewan-ahmed**
# What changed, and why it matters

This PR updates the contributing guide; especially the instructions around screenshots.

I'm not too sure about the line I added on the `gif`s so I'm looking forward to your take on that.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -137,7 +137,11 @@ Use subtitles to break up the article if it's more than a couple of paragraphs,
 Screenshots
 '''''''''''
 
-Screenshots can be helpful, especially where a user might have difficulty finding a particular element on a screen. They don't need a picture of every button they should click during a process, and whenever we make frontend changes to the web console, every screenshot must be updated. Therefore, use screenshots but only when needed.
+Screenshots might get outdated as soon as there's a visual change on the Aiven console. Although they can be helpful, especially where a user might have difficulty finding a particular element on a screen, use screenshots only when the textual instruction is not enough to help our audience.
+
+When adding screenshots, please add them to the appropriate sub-directory under `images`.
```
Comment (lornajane) [inline CONTRIBUTING.rst:142]: can we say what's appropriate here? The images folder reflects the structure of the docs folder, and the image should be in the folder matching the document that refers to it.

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -137,7 +137,11 @@ Use subtitles to break up the article if it's more than a couple of paragraphs,
...
+
+Sometimes, the site cannot render `gif` properly, so it's better to avoid them.
```
Comment (lornajane) [inline CONTRIBUTING.rst:144]: Let's say "We do not use gifs or animation in Aiven docs". We don't need to say why - it's an accessibility mistake, and also we know our very experienced audience tends not to like this sort of thing. There's no technical problem, we just don't do that here.

**Comment by dewan-ahmed** [comment]
Thanks @lornajane ; makes sense

https://github.com/aiven/devportal/pull/1542/commits/b1c6a64b4ca66b48cf27be9ba22fb0ebf2ef0a82 brings in the changes based on your comments

I used literal around gif to avoid Vale spelling issue.

**Review by lornajane — APPROVED** [review]
This adds some excellent details! Thank you :)

**Pull request merged**